hw06-atm/src/main/java/ru/otus/homework/Cell.java

```diff
@@ -0,0 +1,29 @@
+package ru.otus.homework;
+
+/**
+ * Class for ATM's Cell, it consists of banknote's nominal and number of banknotes for this nominal
+ */
+public class Cell {
+    private int nominal;
```

Comment:
Здесь лучше использовать созданный enum
```suggestion
    private DEFAULT_NOMINALS nominal;
```

Но как написал в другом комментарии, DEFAULT_NOMINALS лучше переименовать в Nominal

Reply:
Заменила использование Integer на Enum